pom.xml

```diff
@@ -2988,6 +2988,7 @@
               <test.src.tables>src</test.src.tables>
             </systemProperties>
             <failIfNoTests>false</failIfNoTests>
+            <failIfNoSpecifiedTests>false</failIfNoSpecifiedTests>
```

Comment:
Isn't `failIfNoSpecifiedTests` a substitute for `failIfNoTests`?

Reply:
Hmmm...it look seems not.
https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html
<img width="1225" alt="image" src="https://user-images.githubusercontent.com/15246973/193261014-7113fdcb-9c5f-444d-b48f-dd8fa6b00e36.png">

Reply:
Got it